azurerm/internal/services/mssql/mssql_database_resource.go

```diff
@@ -134,6 +134,18 @@ func resourceArmMsSqlDatabase() *schema.Resource {
 				ValidateFunc:     validation.IsRFC3339Time,
 			},
 
+			"recoverable_database_id": {
```

Comment:
Again, could we make this
```suggestion
			"recover_database_id": {
```

as we are recovering the database id specified.